spec/migrations/20190108135546_rename_websocket_to_remote_console_spec.rb

```diff
@@ -0,0 +1,69 @@
+require_migration
+
+describe RenameWebsocketToRemoteConsole do
+  let(:settings_change_stub) { migration_stub(:SettingsChange) }
+
+  migration_context :up do
+    it "converts the server roles" do
+      setting_changed = settings_change_stub.create!(:key => "/server/role", :value => "database,websocket,reporting")
+      setting_ignored = settings_change_stub.create!(:key => "/server/role", :value => "database,reporting")
+
+      migrate
+
+      expect(setting_changed.reload.value).to eq("database,remote_console,reporting")
+      expect(setting_ignored.reload.value).to eq("database, reporting")
+    end
+
+    it "converts the log levels" do
+      setting_changed = settings_change_stub.create!(:key => "/log/level_websocket", :value => "foo")
+      setting_ignored = settings_change_stub.create!(:key => "/log/level_api", :value => "bar")
+
+      migrate
+
+      expect(setting_changed.reload.key).to eq("/log/level_remote_console")
+      expect(setting_ignored.reload.key).to eq("/log/level_api")
```

Comment:
I feel like these settings changes should be testing the key and value to verify they're still the same.  Someone might have set level_websocket to debug and would want to ensure the renamed key is still debug.

Reply:
Is it enough if I check `setting_changed.value` and `setting_ignored.value`? Or do I have to do some extra querying as well and if yes how?

Reply:
Yes, I think if you add a check for the `.value`, that's more than enough